src/Native/System.Net.Security.Native/CMakeLists.txt

```diff
@@ -30,4 +33,8 @@ target_link_libraries(System.Net.Security.Native
     ${LIBGSS}
 )
 
+include_directories(SYSTEM
+    ${LIBGSS_INCLUDE_DIR}
```

Comment:
Conflits aren't safe.

I don't know what happens (and if it's predictable to say) what happens when a header is available in two given paths like: /usr/include and /usr/pkg/include. The best is to stick to just single implementation of a library.